raster/r.li/r.li.padrange/padrange.c

```diff
@@ -787,8 +787,8 @@ int calculateF(int fd, struct area_entry *ad, double *result)
 	              (((NS_DIST1 + NS_DIST2) / 2) / hd.rows);
 
 	/* get min and max patch size */
-	min = 1.0 / 0.0;	/* inf */
-	max = -1.0 / 0.0;	/* -inf */
+	sscanf("inf", "%lf", &min);	/* inf */
+	sscanf("-inf", "%lf", &max); /* -inf */
```

Comment:
In #2681, the code was changed to use `INFINITY` and `-INFINITY,` but there's a note that it might not for MSVC (at that time):

```c
        min = INFINITY;
        max = -INFINITY;
```